exchange/bidder.go

```diff
@@ -280,27 +280,33 @@ func addNativeTypes(bid *openrtb2.Bid, request *openrtb2.BidRequest) (*nativeRes
 
 func setAssetTypes(asset nativeResponse.Asset, nativePayload nativeRequests.Request) error {
 	if asset.Img != nil {
-		if tempAsset, err := getAssetByID(asset.ID, nativePayload.Assets); err == nil {
+		if asset.ID == nil {
+			return fmt.Errorf("Response Image asset doesn't have an ID")
```

Comment:
Nitpick: There's no need to use `fmt.Errorf` since there is no dynamic content. Please use `errors.New()` here and for the Data asset.

Reply:
Oh yeah of course, didn't notice that. Will fix.